modules/statik/src/variables/StatikVariable.php

```diff
@@ -51,4 +53,14 @@ public function slugify(string $string): string
     {
         return ElementHelper::generateSlug($string);
     }
+
+    public function hyphenate($string, $minimumWordLength = 12)
+    {
+        $language = strtolower(explode('-', Craft::$app->language)[0]);
+        if($language == 'en') $language = 'en-us';
```

Comment:
Dit kan je beter vervangen met 
`$language = $language === 'en' ? 'en-us' : $language;`

PHP core wil op termijn support voor impliciete brackets laten vallen in zo een if.

Reply:
Extra vraag bij `$app->language` - Kunnen we hier ook niet werken met de taal van de huidige site?